webapp/src/components/properties/dateRange/__snapshots__/dateRange.test.tsx.snap

```diff
@@ -43,9 +43,7 @@ exports[`components/properties/dateRange returns default correctly 1`] = `
       class="Button    "
       type="button"
     >
-      <span>
-        Empty
-      </span>
+      <span />
```

Comment:
Is `span` (non-container tag) allowed to open/close in one tag?  Will this snap match the actual HTML?

Reply:
... also it occurred to me that adding something to the span that helps screen readers would be useful.  It would be hidden but read aloud.